package.json

```diff
@@ -28,10 +28,12 @@
     "compile": "webpack --mode none",
     "watch": "webpack --mode development --watch --info-verbosity verbose",
     "test-compile": "tsc -p ./",
-    "postinstall": "node ./node_modules/vscode/bin/install",
-    "test": "node ./node_modules/vscode/bin/test",
-    "build-server": "./node_modules/.bin/gulp build_server",
-    "tslint": "./node_modules/.bin/gulp tslint"
+    "pretest": "npm run test-compile",
+    "test": "node ./out/test/runTest.js",
+    "tslint": "./node_modules/.bin/tslint -p .",
+    "build": "./node_modules/.bin/gulp build",
+    "build-server": "./node_modules/.bin/gulp buildServer",
```

Comment:
`buildServer` should be `build_server` because the gulp task is called `build_server`: https://github.com/angelozerr/vscode-xml/blob/5b1969aa5a1e5c73a08a00f45ffb68d358cfeb43/gulpfile.js#L13

Reply:
alternatively, if you wish, you can rename the  `build_server` task to `buildServer` in the gulpfile

Reply:
Good catch. It was a copy/paste mistake. It should be fixed now.